### PR TITLE
signin page done

### PR DESCRIPTION
**Pull request by Venkatknight2295**
issue #42

### CODE REVIEW

File: frontend/src/Poppins/OFL.txt [hunk 1]
```diff
@@ -0,0 +1,93 @@
+Copyright 2020 The Poppins Project Authors (https://github.com/itfoundry/Poppins)
```
Comment (Satyam1923) [inline frontend/src/Poppins/OFL.txt:1]: keep only the fonts we'll be using.

**Review by Satyam1923 — COMMENTED** [review]
Don't use picture for background try making with css 
don't use icons other than of hugeicons for now

File: frontend/src/Poppins/OFL.txt (hunk 1 above)
Comment (Venkatknight2295) [reply frontend/src/Poppins/OFL.txt:1]: yeah this font is used in your figma file

Comment (Venkatknight2295) [reply frontend/src/Poppins/OFL.txt:1]: as it was used in figma design i used this font please check

Comment (Venkatknight2295) [reply frontend/src/Poppins/OFL.txt:1]: ok changed font to Roboto

**Comment by Venkatknight2295** [comment]
@Satyam1923 making  the background in only css is hard and takes time i can only do if you make this issue level 2 at least

**Comment by Satyam1923** [comment]
Fine

**Pull request closed without merging**